n64.mk

```diff
@@ -98,12 +104,17 @@ $(BUILD_DIR)/%.o: $(SOURCE_DIR)/%.c
 	@echo "    [CC] $<"
 	$(CC) -c $(CFLAGS) -o $@ $<
 
-%.elf: $(N64_ROOTDIR)/mips64-elf/lib/libdragon.a $(N64_ROOTDIR)/mips64-elf/lib/libdragonsys.a $(N64_ROOTDIR)/mips64-elf/lib/n64.ld
+%.elf: $(N64_LIBPATH)/libdragon.a $(N64_LIBPATH)/libdragonsys.a $(N64_LIBPATH)/n64.ld
 	@mkdir -p $(BUILD_DIR)
 	@echo "    [LD] $@"
-	$(LD) -o $@ $(filter-out $(N64_ROOTDIR)/mips64-elf/lib/n64.ld,$^) $(LDFLAGS) -Map=$(BUILD_DIR)/$(notdir $(basename $@)).map
+	$(LD) -o $@ $(filter-out $(N64_LIBPATH)/n64.ld,$^) $(LDFLAGS) -Map=$(BUILD_DIR)/$(notdir $(basename $@)).map
 	$(N64_SIZE) -G $@
 
+%.bin: %.elf
+	@mkdir -p $(dir $@)
+	@echo "    [OBJCOPY] $@"
+	$(N64_OBJCOPY) $< $@ -O binary
```

Comment:
I have a similar concern with this. I don't think the bin has a value per-se to warrant a differ Makefile target. Just like while building rSP
ucode, I'd keep it folded in the N64 step